lib/docparser.js

```diff
@@ -401,7 +402,30 @@ YUI.add('docparser', function (Y) {
                 target[tagname] = result;
 
             },
-            'throws': 'return',
+
+            // @throws {type} description
+            'throws': function (tagname, value, target) {
```

Comment:
Yes. It was deliberate because I have the potential of change `@return` implementation to support returnning complex object (see #37). Anyway, I'd like to do it as you suggest for now.